src/Components/Publishing/Layouts/__tests__/NewsLayoutAdsEnabled.test.tsx

```diff
@@ -44,25 +43,19 @@ describe("News Layout with new ads enabled", () => {
   it("renders the news layout component with new ads component", () => {
     const component = getWrapper()
 
-    expect(component.find(NewDisplayCanvas).length).toBe(1)
-    expect(component.find(DisplayCanvas).length).toBe(0)
+    expect(component.find(DisplayAd).length).toBe(1)
   })
 
   it("renders the news layout component with correct ad unit after the 3rd article on desktop", () => {
     props.articleSerial = 3
     const component = getWrapper()
 
-    expect(component.find(NewDisplayCanvas).length).toBe(1)
-    expect(component.find(DisplayCanvas).length).toBe(0)
+    const ads = component.find(DisplayAd)
```

Comment:
This looks good, only now we can also delete this separate test file `NewsLayoutAdsEnabled.test.tsx` move these new tests to  `NewsLayoutAds.test.tsx`. Also in  `NewsLayoutAds.test.tsx` we should remove all assertions that test the `DisplayCanvas` or `DisplayPanel` components as well as remove any tests in `CanvasFooter` that are used for ads. Note: we used to use the `CanvasFooter` for 970x250 ads AND other components like the related articles rail, and the collections rails, so some `CanvasFooter` components need to remain.